NEWS.md

```diff
@@ -3,38 +3,41 @@
 
 ## Highlights
 
-<Fill this block manually from the blocks below>
+ * `DESTIP/DESTPORT/PROTO`: new macros. ([#2899](https://github.com/syslog-ng/syslog-ng/pull/2899))
+ * `set-facility()`: add new rewrite operation to change the syslog facility associated with the message. ([#3136](https://github.com/syslog-ng/syslog-ng/pull/3136))
+ * `network tls`: Added `ca-file()` option. With this option the user can set a bundled CA-file to verify the peer. ([#3145](https://github.com/syslog-ng/syslog-ng/pull/3145))
```

Comment:
Optional: I think this does not hit the _Highlights_ level, but it is completely fine for me to include here.

Reply:
I think this is one greatly simplifies the configuration of tls. That's why I felt it should be in the highlight's. I would like to keep that one there.

Reply:
Sure :)